apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -77,8 +109,40 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
     sendPythonCodeToMicroBit(pythonCode);
   };
 
+  const onClickSettings = async () => {
+    const results = await dialogControl?.showDialog({
+      type: DialogType.GenericDropdown,
+      title: 'Settings',
+      message: 'Customize your text editor font size',
+      selectedValue: selectedFontSizeKey,
+      items: fontSizeOptions,
+      dropdownLabel: '',
+    });
```

Comment:
When I talked about this with Sam, I imagined it being a dropdown similar to version history, rather than a modal. I think this is fine (certainly to start with), but it would be worth confirming we are good long-term with a modal!

Reply:
https://codedotorg.slack.com/archives/C06JELCAPT9/p1743161842018539?thread_ts=1742576084.787909&cid=C06JELCAPT9

Reply:
Thanks Molly! I just chatted with @samantha-code , and confirmed still would like something similar to the version history dropdown and not a dialog/modal. Converting to draft to re-do UI. Thanks!